GooglePlayInstant/Tests/Editor/AndroidInstantTest.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2018 Google LLC
```

Comment:
I'm debating this, but thoughts on this for path/class name?
Samples/TestApp/Editor/TestAppBuilder.cs

Reply:
I like that better, and it'll be good to move it out of the Tests folder.